src/utils/Chart.js

```diff
@@ -0,0 +1,42 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    let max = d3.max(data, d => {
+      return d[axis];
+    });
+
+    let min = d3.min(data, d => {
+      return d[axis];
+    });
+
+    const maxDigits = Math.ceil(max).toString().length - 1;
+    const multiplier = Math.pow(10, maxDigits);
+
+    max = Math.ceil(max / multiplier) * multiplier;
+    min = Math.floor(min / multiplier) * multiplier;
```

Comment:
In the case of the min value, your example would actually end up being 0 because the min value gets floored.  But I can see who if the min number ended up being a negative number how that could result in a large negative min value.  I'll add a custom min multiplier.